kokudaily/sql/cust_node_report_setup.sql

```diff
@@ -26,30 +26,23 @@ INSERT
       report_month,
       pod_count,
       cpu_cores,
-      memory_bytes
+      memory_gigabytes
   )
 SELECT ''%%1$s'' AS "customer",
        rpp.provider_id AS "provider_id",
        rpp.cluster_id AS "cluster_id",
        ro.node as "node",
-       date_trunc(''month'', rp.interval_start)::date AS "report_month",
-       count(DISTINCT ro.pod) AS "pod_count",
+       date_trunc(''month'', rpp.report_period_start)::date AS "report_month",
+       0 AS "pod_count",
```

Comment:
We can probably remove pod_count entirely and remove it from the CREATE and INSERT statements

Reply:
You don't think `pod_count` might be useful for evaluating some of the first customers to allow list resource optimizations?

Reply:
I do, it's just not a thing in the daily summary table. We'd have to switch to pulling this report from Trino, and probably should!